app/uk/gov/hmrc/gform/auditing/AuditService.scala

```diff
@@ -64,26 +64,26 @@ trait AuditService {
     processedData.map(x => x.id.value -> x.value).toMap
   }
 
-  def sendSubmissionEvent(form: Form, sections: List[BaseSection], retrievals: Retrievals)(implicit ec: ExecutionContext, hc: HeaderCarrier, request: Request[_]) = {
-    sendEvent(form, formToMap(form, sections), retrievals)
+  def sendSubmissionEvent(form: Form, sections: List[BaseSection], retrievals: Retrievals, customerId: String)(implicit ec: ExecutionContext, hc: HeaderCarrier, request: Request[_]) = {
```

Comment:
This `customerId` is great candidate for lifting into value class, so we avoid working with `String` type. I'd be happy to see this change, but it does not necessary needs to be part of this PR.

Reply:
Agreed, created https://jira.tools.tax.service.gov.uk/browse/GFC-337